docs/STREAMING.md

```diff
@@ -0,0 +1,75 @@
+# Streaming 
+
+
+Uses grpc streams to allow the plugin to send data immediately or after a certain period of time instead of on an interval governed by Snap.
```

Comment:
Can you edit this to start with, `Streaming plugins use grpc streams to allow ...`